Tests/Twig/Extension/NumberExtensionTest.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+/*
+ * This file is part of the Sonata project.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ */
+
+namespace Sonata\IntlBundle\Tests\Twig\Extension;
+
+use Sonata\IntlBundle\Templating\Helper\NumberHelper;
+use Sonata\IntlBundle\Twig\Extension\NumberExtension;
+
```

Comment:
i would like to have you with ``@author`` here, same in other new classes, if there are some